examples/react-renderer/src/index.css

```diff
@@ -12,3 +12,7 @@ body {
   -moz-osx-font-smoothing: grayscale;
   padding: 1rem 0;
 }
+
+strong {
+  font-weight: bold;
+}
```

Comment:
this isn't the default?

Reply:
We had a `reset.css` file so it's wasn't the default anymore. But I removed it in 879f2421440e37bebc08e250dd36e4564bdfb460 so that we use the same stylesheet in all our examples.